packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewGroupDrawingOrderHelper.java

```diff
@@ -65,6 +69,20 @@ public boolean shouldEnableCustomDrawingOrder() {
    * ViewGroup#getChildDrawingOrder}.
    */
   public int getChildDrawingOrder(int childCount, int index) {
+    if (mDrawingOrderIndices != null
+      && (index >= mDrawingOrderIndices.length || mDrawingOrderIndices[index] >= childCount)) {
+      FLog.w(ReactConstants.TAG,
+        "getChildDrawingOrder index out of bounds! please check your operations of addding and " +
```

Comment:
FLog takes a format string. Let's also not make this too verbose, you can always attach a debugger to inspect the full state.

```suggestion
      FLog.w(ReactConstants.TAG,
        "getChildDrawingOrder index out of bounds! Please check any costom view manipulations you may have done. childCount = %d, index = %d", ..)
```